### PR TITLE
Create test file for the Seeding Input

### PR DESCRIPTION
**Pull request by jamesng5**
__Pull Request Description__

Create new test file for SeedingInput #160 

---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <body>
     <div id='seedingInput'>
-        <h1 style='text-align:center;'>Seeding Input Log</h1> 
+        <h1 data-cy="header" style='text-align:center;'>Seeding Input Log</h1>
```
Comment (johnmaccormick) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.html:3]: Would  more unique name be better here?

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    beforeEach(()=>{
+        cy.login("manager1", "farmdata2")
+        cy.visit('/farm/fd2-field-kit/seedingInput')
+    })
+
+    it("Check that the SUBMIT button is disabled", () => {
+        cy.get("[data-cy=submit-button]").should("be.disabled")
+    })
+
+    it('Test the comment text box is empty and enable', ()=>{
```
Comment (johnmaccormick) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js:11]: typo -- "enabled"

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    })
+
+    it('Test the Submit button box is labeled', ()=>{
+        cy.get("[data-cy=submit-button]").should("have.text","Submit").should('be.visible')
+    })
+    
+    it("Check that the page has a header", () => {
+        cy.get("[data-cy=header]").should("have.text", "Seeding Input Log")
+    })
+    it("Check that the section has a label comment", () => {
```
Comment (johnmaccormick) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js:23]: use consistent line spacing

**Review by johnmaccormick — CHANGES_REQUESTED** [review]
nice! -- minor suggestions for changes

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        cy.get("[data-cy=comments]").should("be.visible")     
+    })
+})
```
Comment (johnmaccormick) [inline farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js:26]: have you checked that the label is "Comments"?

Comment (johnmaccormick) [reply farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.misc.defaults.spec.js:26]: Just clarifying, note that one of the tasks in Issue #160 is: 'has a section with the label "Comments"' -- so you should really check the label itself not just the visibility.

File: farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.html (hunk 1 above)
Comment (jamesng5) [reply farmdata2/farmdata2_modules/fd2_field_kit/seedingInput/seedingInput.html:3]: This name is from the original file, and we did not add or make any changes in seedingInput.html file.

**Comment by johnmaccormick** [comment]
@jamesng5 @vuphuongha @tainguyen103 -- The final test is failing. See if you find the problem on your own. If not, let me know and I will give you a hint. 

Other things to work on:
* make sure you have consistent spacing between the it() blocks
* put a comment at the top of the file explaining the purpose of this file

**Comment by tainguyen103** [comment]
@johnmaccormick Professor MacCormick, are you referring to the "Check that the submit button is disabled"? I tried running the test on my machine and it works fine. 

Screenshot 2023-04-24 at 8 27 17 PM

**Comment by jamesng5** [comment]
> @jamesng5 @vuphuongha @tainguyen103 -- The final test is failing. See if you find the problem on your own. If not, let me know and I will give you a hint.
> 
> Other things to work on:
> 
> * make sure you have consistent spacing between the it() blocks
> * put a comment at the top of the file explaining the purpose of this file

I have done working on this. Please let me know if any further changes needed.

**Comment by johnmaccormick** [comment]
@tainguyen103 @vuphuongha @jamesng5  --

The final test that looks for the "Comment" is still failing for me. Here is what I see:

image

can you confirm that this test is *not* failing for you?

**Comment by vuphuongha** [comment]
Screenshot 2023-04-27 at 7 52 16 PM
Not sure what happened, but it worked fine on my end. Should I check with you again during tomorrow's office hours? @johnmaccormick

**Comment by braughtg** [comment]
> The final test that looks for the "Comment" is still failing for me.

I see this test as failing on my system as well.

image

**Comment by johnmaccormick** [comment]
nice work, this is now being merged!

**Pull request merged**